primitives/src/const_vec.rs

```diff
@@ -92,6 +92,33 @@ impl<T: IntoSchema> IntoSchema for ConstVec<T> {
     }
 }
 
+impl<T> IntoIterator for ConstVec<T> {
+    type Item = T;
+
+    type IntoIter = <Vec<T> as IntoIterator>::IntoIter;
+
+    fn into_iter(self) -> Self::IntoIter {
+        self.into_vec().into_iter()
+    }
+}
+
+/// Trait to extend `[T]` with a method to convert it to `ConstVec<T>` by analogy with `[T]::to_vec()`.
```

Comment:
why can't it be inherent impl?

Reply:
Not sure what do you mean here. Isn't it the most generic implementation?

Reply:
why did you make a trait that is implemented by only 1 type?

Reply:
You add methods to types that are defined not in your crate (it's in std in this case).

This is a so-called extension trait, a [common pattern in rust](https://rust-lang.github.io/rfcs/0445-extension-trait-conventions.html)

Reply:
yeah, sorry, I thought it was implemented on `ConstVec`. Now I see